package.json

```diff
@@ -39,6 +39,7 @@
         "d3-time": "^3.0.0",
         "d3-time-format": "^4.0.0",
         "danger": "^11.2.3",
+        "eslint-plugin-jsdoc": "^44.0.1",
```

Comment:
This doesn't need to be checked in. I found it useful during development to turn the rule on & off.

```suggestion
```